WordPress/src/main/res/values-night/styles.xml

```diff
@@ -124,4 +124,58 @@
         <item name="android:background">?android:attr/listDivider</item>
     </style>
 
+    <!-- Zendesk Styles Override for Dark Theme -->
+
+    <style name="ZendeskActivityDefaultTheme" parent="WordPress.NoActionBar">
+        <item name="colorPrimary">#282828</item>
+        <item name="snackbarButtonStyle">@style/ZendeskSdkTheme.Snackbar.Button</item>
+        <item name="snackbarStyle">@style/Widget.MaterialComponents.Snackbar</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.RequestList.AvatarText" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">14sp</item>
+        <item name="android:lines">1</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.Request.Message.SystemMessage" tools:ignore="UnusedResources" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:lineSpacingExtra">8dp</item>
+        <item name="android:fontFamily">sans-serif</item>
+        <item name="android:textStyle">normal</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.RequestList.Subject" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">14sp</item>
+        <item name="android:lines">1</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.RequestList.Title" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textSize">16sp</item>
+        <item name="android:lines">1</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.Request.Date" parent="@style/TextAppearance.MaterialComponents.Body1">
+        <item name="android:textColor">?attr/wpColorOnSurfaceMedium</item>
+        <item name="android:textSize">14sp</item>
+        <item name="android:fontFamily">sans-serif</item>
+        <item name="android:textStyle">normal</item>
+    </style>
+
+    <style name="ZendeskRequestListButton" parent="@style/WordPress.Button.Primary" />
+
+    <style name="ZendeskSdkTheme.Snackbar.Button" tools:ignore="UnusedResources" parent="@style/Widget.AppCompat.Button.Borderless">
+        <item name="android:textColor">@color/blue_30</item>
+    </style>
+
+    <style name="ZendeskSdkTheme.Light.Request.Dialog.Retry" parent="ZendeskSdkTheme.Light">
+        <item name="android:clickable">true</item>
+        <item name="android:focusable">true</item>
+        <item name="android:foreground">?selectableItemBackground</item>
+        <item name="android:paddingBottom">12dp</item>
+        <item name="android:paddingTop">12dp</item>
+        <item name="android:paddingEnd">16dp</item>
+        <item name="android:paddingRight">16dp</item>
+        <item name="android:paddingStart">16dp</item>
+        <item name="android:paddingLeft">16dp</item>
+    </style>
+
```

Comment:
Same as above...this should also be moved to the main styles resource file since it doesn't specify anything specific to dark mode.